src/Navigation/Link/styled.js

```diff
@@ -42,15 +42,14 @@ export const StyledNavLink = styled(NavLink).attrs(() => ({ activeClassName }))`
   padding: 14px 16px;
```

Comment:
In this file I see one more error (we could forget about it before) there is missing curly bracket after
`@media (max-width: ${({ theme }) => theme.breakpoints.mobile}px) {
    padding: 0;
    gap: 8px;
`;`
please add it when you make corrections